src/librustc_mir/interpret/mod.rs

```diff
@@ -1,5 +1,10 @@
 //! An interpreter for MIR used in CTFE and by miri
 
+#[macro_export]
+macro_rules! err {
+    ($($tt:tt)*) => { Err($crate::interpret::EvalErrorKind::$($tt)*.into()) };
+}
```

Comment:
What about moving this to error.rs? I think it'd make more sense to have it there.